pkg/config/optimizely_config.go

```diff
@@ -23,12 +23,15 @@ import (
 
 // OptimizelyConfig is a snapshot of the experiments and features in the project config
 type OptimizelyConfig struct {
-	Revision       string                          `json:"revision"`
-	ExperimentsMap map[string]OptimizelyExperiment `json:"experimentsMap"`
-	FeaturesMap    map[string]OptimizelyFeature    `json:"featuresMap"`
-	SdkKey         string                          `json:"sdkKey,omitempty"`
-	EnvironmentKey string                          `json:"environmentKey,omitempty"`
-	datafile       string
+	Revision            string                          `json:"revision"`
+	ExperimentsMap      map[string]OptimizelyExperiment `json:"experimentsMap"`
+	FeaturesMap         map[string]OptimizelyFeature    `json:"featuresMap"`
+	SdkKey              string                          `json:"sdkKey,omitempty"`
```

Comment:
@jaeopt Should I allow an empty string here? The doc says I should. If I remove "omitempty", it will output the sdkKey. What about environmentKey?

Reply:
For all SDKs, let's keep empty string for both sdkKey and environmentKey when no valid values are available for them.

Reply:
Thanks. I'll remove the `omitempty`.